docs/cell_census_storage_and_release_policy.md

```diff
@@ -0,0 +1,82 @@
+# Cell Census storage & release policy
+
+**Status**: Draft.
+
+**Version**: 0.0.1.
+
+**Last edited**: Dec, 2022.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14), [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+---
+
+The cell census MUST be stored in the following S3 root bucket:
+
+`s3://cellxgene-data-public/`
+
+Each release MUST be deposited in a folder under the root that follows this naming convention. `counter` MUST be an integer indicating the release number starting on `1`:
```

Comment:
You may want to make it clear that unofficial builds (ie., not a capital-R "RELEASE") can use any naming scheme.  I am confident we will need, for example, test releases, release candidates, etc.  All which need to be stuffed into the `release.json` machinery - ie, the software must support them.

So, I'd suggest something like:
* Census "build" is a SOMA collection / container containing cell data.
* Census "RELEASE" is a "build" which has been officially blessed and released in a formal manner.
* A "build" is named with a `tag`, which is a string of printable ASCII (eg, "foo", "2022-11-01", "1-rc2.9")
* a RELEASE is named with a tag following the conventional format `release-[counter]` 

this lets you publish release candidates using tag names like `release-1-rc1` or `release-1-for-validation`, etc.